deploy/modules/gcp/tidb-cluster/main.tf

```diff
@@ -41,6 +41,10 @@ resource "google_container_node_pool" "tikv_pool" {
   name       = "${var.cluster_name}-tikv-pool"
   node_count = var.tikv_node_count
 
+  depends_on = [
+    var.cluster_id
+  ]
```

Comment:
`tikv_pool` is the first resource to create in this module. when it depends on a cluster-ID, it will depend on the module `tidb-operator` which creates the cluster.